search/editor_delegate.cpp

```diff
@@ -0,0 +1,44 @@
+#include "search/editor_delegate.hpp"
+
```

Comment:
Not sure. The first line for cpp implementation for header should (from my point of view) be an inclusion of the header, separated from other includes by blank line.